frontend/public/components/monitoring.tsx

```diff
@@ -1071,6 +1073,35 @@ export const MonitoringUI = () => <Switch>
   <Route component={PollerPages} />
```

Comment:
I think we need to add one more `Route` here:

```<Route path="/monitoring/(alertmanageryaml|alerts|silences)" exact render={() => <AlertingPage match={match} />} />```

The `match` prop is already being passed to `MonitoringUI`.

Then the monitoring UI will be handling its own routing and all of the changes in app-contents.tsx should become unnecessary.

Reply:
Hi @kyoto, I had to add those Routes to app-contents.tsx in order for the `HorizontalNav` to work, but maybe with the SimpleTabs I won't need them.

Reply:
Hi, I still needed to add 3 routes to app-contents.tsx for the three tab pages in order for the match URL to work, else it always was just `/monitoring`.

Reply:
Hmmmm. React Router also passes a `location` prop, so you could use `location.pathname` instead of `match`, which should then remove the need for the new `app-contents.tsx` routes.

Reply:
Not sure how to use `location.pathname` for a `<Route path='...' />` match?